src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1299,6 +1299,36 @@ private class ACCT_IndividualAccounts_TEST {
         Test.stopTest();
     }
 
+    /*********************************************************************************************************
+    @description 
+        Test Contact insert and update when Household Naming Settings is invalid
+        and the Automatic Household Naming is turned off 
+    verify:
+        The Contact is created and updated successfully
+    **********************************************************************************************************/ 
+    private static testMethod void contactDmlShouldSucceedWhenHHNamingSettingsIsInvalidAndAutomaticHouseholdNamingIsTurnedOff() {
+        turnOffAutomaticHHNaming();
```

Comment:
These tests not only verify features but methods as well. When a test is selected (via Developer Console) it is beneficial to know what is being tested and what expected behaviour is (comments specified above tests in the Apex class are not visible). The clear name is also beneficial when test fail. Moreover, there are many tests verifying the same method on different conditions, so it is important for the name to indicate difference. 

I usually try to name test as _featureOrMethodTestedShouldProduceResultWhenConditions_. I am open to any test naming convention that is not verbose and clearly explains what is being tested and expected.